test/test.html

```diff
@@ -6,6 +6,8 @@
         <script src="jasmine/boot.js"></script>
         <link rel="stylesheet" href="jasmine/jasmine.css">
         <script src="katex-spec.js"></script>
+        <script src="error-spec.js"></script>
+        <script src="unicode-spec.js"></script>
```

Comment:
The way I see it, these files [should get concatenated by `server.js`](https://github.com/Khan/KaTeX/blob/befe1c1af7a2ce88a1c7099ee260f05d834d76c6/server.js#L52) since #389, so this should not be needed. Or did you actually encounter a situation where it *was* needed? If not, should we aim for different names for the `katex-spec.js` as contained in the repository and the combined one served by the server? Which one to rename? I think `katex-spec.js` makes a lot of sense for the combined version, so `main-spec.js` or similar for the single file might be better.